src/readii/cli/main.py

```
@@ -0,0 +1,35 @@
+import click
+
+from orcestradownloader.managers import DatasetRegistry, DatasetManager, UnifiedDataManager
```

Ruff (0.8.2)

`orcestradownloader.managers.UnifiedDataManager` imported but unused

Remove unused import: `orcestradownloader.managers.UnifiedDataManager`

(F401)